examples/artemis-data-binding/artemis_data_binding.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/artemis;
+import ballerina/http;
+import ballerina/log;
+
+type Order record {
+    string description?;
+    int id;
+    float cost;
+};
+
+// Consumer listens to the queue (i.e., "my_queue") with the address
+// (i.e., "my_address").
+@artemis:ServiceConfig {
+    queueConfig: {
+        queueName: "my_queue",
+        addressName: "my_address"
+    }
+}
+// Attaches the service to the listener.
+service artemisConsumer on new artemis:Listener(
+                            {host:"localhost", port:61616}) {
```

Comment:
```suggestion
                            { host: "localhost", port: 61616 }) {
```

Reply:
I made these changes but VSCode doesn't seem to format it this way. Is it an issue?

Reply:
Created an issue https://github.com/ballerina-platform/ballerina-lang/issues/15373